policykit/slackintegration/views.py

```diff
@@ -79,42 +81,54 @@ def oauth(request):
                     )
                 user_group.community = community
                 user_group.save()
-
-                cg = CommunityDoc.objects.create(text='',
-                                                 community=community)
-
-
+                    
+                cg = CommunityDoc.objects.create(text='', community=community)
+                    
                 community.community_guidelines=cg
                 community.save()
 
-            else:
-                community = s[0]
-                community.community_name = res['team']['name']
-                community.team_id = res['team']['id']
-                community.access_token = res['access_token']
-                community.save()
-
-            user = SlackUser.objects.filter(username=res['authed_user']['id'])
+                #get the list of users, create SlackUser object for each user
+                data2 = parse.urlencode({
+                    'token':community.access_token
+                }).encode()
+                                       
+                req2 = urllib.request.Request('https://slack.com/api/users.list', data=data2)
+                resp2 = urllib.request.urlopen(req2)
+                res2 = json.loads(resp2.read().decode('utf-8'))
+                
+                #https://api.slack.com/methods/users.list
```

Comment:
just checking, @ekim6767 - is users.list an admin-only API request? if so, then we don't even need the below call to users.info that checks if it returns an admin error, can just use the result from this call?

Reply:
I believe that it works with the user, bot and admin tokens, and i'm using the bot token in the code, but I think there is a way to integrate the user.info call with it

Reply:
ah well if there's not a way to do check/determination for adminship using users.list then let's forget about itt.